common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -78,7 +79,7 @@
                             @fragments.inlineSvg("search-36", "icon", List("main-menu__icon", "main-menu__icon--search"))
                         </label>
 
-                        <label for="q"
```

Comment:
Yeah - the name is the same. It's just the hidden label, which currently is disconnected to the input field, because the id is not set.